core/pom.xml

```diff
@@ -87,6 +87,12 @@
       <version>3.3.0.0-alpha.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>4.1.0</version>
```

Comment:
```suggestion
      <version>${mockito.version}</version>
```

Reply:
Mockito was added by someone else. It needs to be in the Pom or IntelliJ will not run any tests because snowflake uses it.